scripts/change_url

```diff
@@ -77,13 +77,6 @@ then
 	change_path=1
 fi
 
-#=================================================
-# MANAGE FAILURE OF THE SCRIPT
-#=================================================
-
-# Exit if an error occurs during the execution of the script
-ynh_abort_if_errors
```

Comment:
Why was there a `ynh_abort_if_errors` here?

Reply:
A duplicate, this second one is now useless since we have ynh_backup_before_upgrade before in the script.

Reply:
Yup I saw that :-)